pkg/cmd/task/list.go

```diff
@@ -66,40 +90,56 @@ func listCommand(p cli.Params) *cobra.Command {
 				Out: cmd.OutOrStdout(),
 				Err: cmd.OutOrStderr(),
 			}
-			return printTaskDetails(stream, p)
+			return printTaskDetails(stream, p, opts.AllNamespaces, opts.NoHeaders)
 		},
 	}
 	f.AddFlags(c)
+	c.Flags().BoolVarP(&opts.AllNamespaces, "all-namespaces", "A", opts.AllNamespaces, "list tasks from all namespaces")
+	c.Flags().BoolVarP(&opts.NoHeaders, "no-headers", "", opts.NoHeaders, "do not print column headers with output (default print column headers with output)")
 
 	return c
 }
 
-func printTaskDetails(s *cli.Stream, p cli.Params) error {
+func printTaskDetails(s *cli.Stream, p cli.Params, allnamespaces bool, noheaders bool) error {
 	cs, err := p.Clients()
 	if err != nil {
 		return err
 	}
 
-	tasks, err := task.List(cs, metav1.ListOptions{}, p.Namespace())
+	ns := p.Namespace()
+	if allnamespaces {
+		ns = ""
+	}
+	tasks, err := task.List(cs, metav1.ListOptions{}, ns)
 	if err != nil {
-		fmt.Fprintf(os.Stderr, "Failed to list tasks from %s namespace \n", p.Namespace())
+		fmt.Fprintf(os.Stderr, "Failed to list tasks from %s namespace \n", ns)
```

Comment:
It would be really nice if we start error messages in lower case

Reply:
Updated